.gitignore

```diff
@@ -59,3 +59,7 @@ generated
 
 # local cached models
 models
+
+# gstsavantframemeta specific
+libs/gstsavantframemeta/_skbuild
+libs/gstsavantframemeta/dist
```

Comment:
`dist` is ignored already

let's add `_skbuild` (just _skbuild) to `# Distribution / packaging` section